civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -494,6 +494,7 @@ function civihr_employee_portal_init() {
   // calling this inside hook_init so we could access
   // some drupal functions such as user_access
   _user_redirection();
+  _civihr_employee_portal_set_up_google_analytics_data_layer();
```

Comment:
@AkA84 hook_init() is also called from the command line (for example, when you're running drush commands for the site). In that context, a few things used by this function might not be available (a logged in user, for example). Could you please try running a few drush commands from a site pointing to this branch and see if you will get any errors? I suggest you try things like, `drush uli`, `drush pm-list`, `drush cc`, `drush fr`

Reply:
that's a good point, thanks. I tried several `drush` commands on my local and got no error, seems to be working fine